index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav class="nav-bar" id="navbar">
+        <header>Web Developer</header>
+        <ul>
+          <li><a class="nav-link" href="#Skills">Skills</a></li>
+          <li><a class="nav-link" href="#Cooperative">Cooperative</a></li>
+          <li><a class="nav-link" href="#Fast_learn">Fast learn</a></li>
+          <li><a class="nav-link" href="#Logical">Logical</a></li>
+          <li>
+            <a class="nav-link" href="#Positive_attitude">Positive attitude</a>
+          </li>
+        </ul>
+      </nav>
+      <section class="main-section" id="Skills">
+        <header>Skills</header>
+        <p></p>
+
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
+      </section>
+      <section class="main-section" id="Cooperative">
+        <header>Cooperative</header>
+        <p></p>
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
+      </section>
+      <section class="main-section" id="Fast_learn">
+        <header>Fast learn</header>
+        <p></p>
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
+      </section>
+      <section class="main-section" id="Logical">
+        <header>Logical</header>
+        <p></p>
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
+      </section>
+      <section class="main-section" id="Positive_attitude">
+        <header>Positive attitude</header>
+        <p></p>
+        <p></p>
+        <code></code>
+        <ul>
+          <li></li>
+        </ul>
+      </section>
+    </main>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
remove all unnecessary comments

Reply:
- [ ] Good job so far, :+1:  I think you can improve this message by adding a short explanation, a "why", the requested change needed to be made. This will make your comment more professional. :smile_cat: